sdk/resourcemanager/Azure.ResourceManager/src/ManagementGroup/Generated/Models/CreateManagementGroupData.cs

```diff
@@ -12,12 +12,12 @@
 namespace Azure.ResourceManager.ManagementGroups.Models
 {
     /// <summary> Management group creation parameters. </summary>
-    public partial class CreateManagementGroupOptions
+    public partial class CreateManagementGroupData
```

Comment:
We should not use the suffix Data unless it inherits from one of our ResourceData objects, we can just undo this rename

Reply:
[Entity][Method]Options -> reserved for optional parameter bag